conf/app.routes

```diff
@@ -4,11 +4,11 @@ GET       /                                           @controllers.Default.redir
 
 GET       /check-eligibility-page                     @uk.gov.hmrc.helptosavestridefrontend.controllers.StrideController.getEligibilityPage()
 
-POST      /customer-eligible                          @uk.gov.hmrc.helptosavestridefrontend.controllers.StrideController.checkEligibilityAndGetPersonalInfo()
+POST      /check-eligibility                          @uk.gov.hmrc.helptosavestridefrontend.controllers.StrideController.checkEligibilityAndGetPersonalInfo()
 
-GET       /you-are-not-eligible                       @uk.gov.hmrc.helptosavestridefrontend.controllers.StrideController.youAreNotEligible()
+GET       /customer-eligible                          @uk.gov.hmrc.helptosavestridefrontend.controllers.StrideController.customerEligible()
 
-GET       /you-are-eligible                           @uk.gov.hmrc.helptosavestridefrontend.controllers.StrideController.youAreEligible()
+GET       /customer-not-eligible                      @uk.gov.hmrc.helptosavestridefrontend.controllers.StrideController.customerNotEligible()
```

Comment:
The requested URL in the ticket is /not-eligible

https://jira.tools.tax.service.gov.uk/browse/HTS-741